src/events/eventBus/EventBusConfig.ts

```diff
@@ -22,14 +32,16 @@ const payloadBuilders = {
   /**
    * Generate event payload to send to event bus when an approved item
    * is scheduled to go on New Tab
+   * @param eventType
    * @param data ScheduledCorpusItemPayload
    * @returns ScheduledItemEventBusPayload
    */
   scheduledItemEvent(
+    eventType: string,
     data: ScheduledCorpusItemPayload
   ): ScheduledItemEventBusPayload {
-    const eventPayload: ScheduledItemEventBusPayload = {
-      eventType: config.eventBridge.addScheduledItemEventType,
+    return {
```

Comment:
just note:  we are re-using the entire body of the `add`, but we will be really consuming only scheduled item externalIds.

when a scheduled item gets deleted, do we also get other details like title, etc - are they all mandatory fields in the payload?

Reply:
we do from the mutation